docs/src/main/asciidoc/reactive-routes.adoc

```diff
@@ -336,7 +336,7 @@ String helloSync(RoutingContext context) {
 ----
 
 Be aware, the processing must be **non-blocking** as reactive routes are invoked on the IO Thread.
-Otherwise, use the `blocking` attribute of the `@Route` annotation or the `@Blocking` annotation.
+Otherwise, set the `type` attribute of the `@Route` annotation to `Route.HandlerType.BLOCKING`, or use the `@Blocking` annotation.
```

Comment:
```suggestion
Otherwise, set the `type` attribute of the `@Route` annotation to `Route.HandlerType.BLOCKING`, or use the `@io.smallrye.common.annotation.Blocking` annotation.
```
We have several of them so better use the FQCN.

Reply:
I'll check all the guide as this annotation is cited multiple times

Reply:
OK, this was the only one.
I updated the guide